pontos/github/api.py

```diff
@@ -362,14 +374,14 @@ def pull_request_files(
         # possible to receive 100
         # might add the page parameter, to get the files 101-202 and so on
         params = {"per_page": "100"}
-        api = f"/repos/{repo}/pulls/{pr_number}/files"
+        api = f"/repos/{repo}/pulls/{pull_request}/files"
         response = self._request(api, params=params)
         file_dict = {}
         for status in status_list:
-            file_dict[status] = [
+            file_dict[status.value] = [
```

Comment:
I am really sure we want to use the `FileStatus` here
```suggestion
            file_dict[status] = [
```